### PR TITLE
Added OAUTH_SAML_LABEL environment variable to allow customization.

### PR DESCRIPTION
**Pull request by xueshanf**
Override the default sign in label "Our SAML Provider", to make it easier for end users. For example: OAUTH_SAML_LABEL=Company ID.

### CODE REVIEW

File: assets/runtime/functions [hunk 1]
```diff
@@ -327,6 +327,7 @@ gitlab_configure_oauth_google() {
   if [[ -n ${OAUTH_GOOGLE_API_KEY} && -n ${OAUTH_GOOGLE_APP_SECRET} ]]; then
     echo "Configuring gitlab::oauth::google..."
     OAUTH_ENABLED=${OAUTH_ENABLED:-true}
+    exec_as_git sed -i 's/{{OAUTH_SAML_LABEL}}/'"${OAUTH_SAML_LABEL}"'/' ${GITLAB_INSTALL_DIR}/config/gitlab.yml
```
Comment (sameersbn) [inline assets/runtime/functions]: @xueshanf this is in the wrong code block. Should be in `gitlab_configure_oauth_saml` function.

**Comment by sameersbn** [comment]
Also could you document `OAUTH_SAML_LABEL` in the README?

File: assets/runtime/functions (hunk 1 above)
Comment (xueshanf) [inline assets/runtime/functions]: Sorry I cut-pasted in wrong place. I will redo it. Please close this one.

**Comment by sameersbn** [comment]
There is no need to close this PR. You can amend to the commit `git cm --amend` and force push `git push -f ...` and then PR will get updated.

**Comment by xueshanf** [comment]
Oh, I was too quick. I removed the old fork,  forked a new one and updated README. I thought it'd be cleaner. I know better next time!

**Comment by sameersbn** [comment]
no problemo.

**Pull request closed without merging**